### ISSUE TITLE
Confusing time display format in pandas DataFrame

### ISSUE THREAD

**Issue by danielballan** [NSLS-II/databroker#59]
attn @yugangzhang

Tom reports that the timezones coming out of the databroker at CHX are not correct. I wrote this simple script to investigate. It prints the local time, runs a dummy scan (no real hardware involved) and shows the table with timestamps. I find that the times agree.

I get this output:

```
Current Time: 2016-05-25 21:03:32.966762-04:00
                              time       det  motor
1 2016-05-25 21:03:33.645711-04:00  0.606531    1.0
2 2016-05-25 21:03:33.703810-04:00  0.135335    2.0
3 2016-05-25 21:03:33.765744-04:00  0.011109    3.0
```

As I run this, it is 5:03 PM in New York. The timestamps show GMT (Greenwich Mean Time) with a `-04:00` that denotes the adjustment to local time, which is set to 'US/Eastern' in metadatastore's configuration.

I am guessing that the times are correct for you as well, but that the display is confusing -- what with the `-04:00` off to the side. It seems we should just display the local time instead, not GMT - 04:00.

``` python
from bluesky import RunEngine
from databroker import db
from metadatastore.commands import insert
from bluesky.plans import scan
from bluesky.examples import motor, det
import time
import pandas as pd


# Setup a RunEngine with metadatastore.
RE = RunEngine({})
RE.subscribe('all', insert)

# Print the local time, for comparison.
secs = time.time()  # seconds since 1 January 1970
localized_time = pd.to_datetime(secs, unit='s').tz_localize('US/Eastern')
print('Current Time:', localized_time)

# Show timestamps from a simple scan.
RE(scan([det], motor, 1, 3, 3))
print(db.get_table(db[-1]))
```

**Comment by yugangzhang** [comment]
Yes, this is exactly the cofusuing time dispay. I agree that we can simply change GMT to local time in the source code.

**Comment by danielballan** [comment]
OK, good. Then we have established there is no bug, but that the display format given by pandas DataFrames is not desirable.

I spent an hour reading about timezone-handling in pandas and numpy. If you didn't know, it's actually a subtle subject. One important complication: the timezone offset changes during the year due daylight savings time, so computing the local time is not a simple subtraction.

I'm not sure it's possible to persuade numpy/pandas/pytz to do what we have in mind here, and I'm loath to devise some hacked solution that might create more confusion for future users. If anyone wants to take a second look at this, feel free to jump in.
